airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -300,7 +365,7 @@ class CRMObjectStream(Stream):
 
     entity: Optional[str] = None
     associations: List[str] = []
-    updated_field = "updatedAt"
+    updated_at_fields = ["updatedAt", "createdAt"]
```

Comment:
why are we doing this? this is really dangerous because we are not using a consistent field for the cursor value. This means we could miss records.

Reply:
we use fallback because `updatedAt` can be null if there is no updates yet

Reply:
nvm -- thought more about it. this makes sense. created_at is always <= updated_at, and as long as we only persist state at the very end of syncing a stream then we should be fine